test/yjsEditorTest.ts

```diff
@@ -0,0 +1,156 @@
+import { Editor, Operation } from 'slate';
+import * as Y from 'yjs';
+import { applySlateOps as applySlateOperations } from '../src/apply';
+import { toSlateOps } from '../src/convert';
+import { SyncDoc, SyncElement } from '../src/model';
+
+export interface YjsEditorTest extends Editor {
+  isRemote: boolean;
+  doc: Y.Doc;
+  syncDoc: SyncDoc;
+  shouldCaptureYjsUpdates: boolean;
+  capturedYjsUpdates: Uint8Array[];
+  onChangeComplete: () => void;
+}
+
+export const YjsEditorTest = {
+  /**
+   * Apply slate ops to Yjs
+   */
+  applySlateOpsToYjs: (e: YjsEditorTest, operations: Operation[]) => {
+    try {
+      e.doc.transact(() => {
+        applySlateOperations(e.syncDoc, operations);
+      });
+    } catch (e) {
+      console.error(e);
+    }
```

Comment:
I would not put a try-catch around this since tests should fail in the case that applySlateOperations throws and exception.

Reply:
Done.